### DISCUSSION TITLE
打开文章后文章反应计数的显示不正确

### DISCUSSION THREAD

**Discussion by ghost** [walinejs/waline#1572] [Q&A] [answered]
问题站点：https://yuanmoe.com
使用主题：vuepress-theme-hope
站点仓库地址：https://github.com/yuanmoe/blog
复现方法：从主页选择一篇文章，点击进入，此时显示的文章反应计数不是正确的
![54F5D98B-DE9C-4262-AC3F-6093856BDE65](https://user-images.githubusercontent.com/48853590/200999423-1d4721d1-7a22-47de-a197-0d8825db8de5.jpeg)
刷新当前页面能够显示的是正确计数
![DCD932DD-FA9B-4BFF-B733-845A6D1DC2C0](https://user-images.githubusercontent.com/48853590/200999478-df6b569a-e072-423a-9f82-8cac14333dd1.jpeg)
我看了一下Waline官网用的也是vuepress-theme-hope 看了一下没这个问题
跪求大佬

**Comment by ghost** [comment]
有人吗有人吗有人吗

**Comment by lizheming** [comment, marked as answer]
`@waline/client` 最新版已修复，感谢反馈。
